### DISCUSSION TITLE
How to change contents of aria-label in Button of day

### DISCUSSION THREAD

**Discussion by jpaulebenz** [gpbl/react-day-picker#1585] [Support] [answered]
How to change contents of aria-label in Button of day
`
<button name="day" class="rdp-button_reset rdp-button rdp-day" aria-label="30th March (Wednesday)" tabindex="0" type="button">
.....
</button>`

I would like to change the format of the "aria-label" of above code to

`
<button name="day" class="rdp-button_reset rdp-button rdp-day" aria-label="30th March (Wednesday) June 2022" tabindex="0" type="button">
.....
</button>`
If there any provisions available in library to do the change.
Any suggestions would be of great help.!

**Comment by gpbl** [comment, marked as answer]
~If you pass `labels` with a `labelDay` function, you can change that `aria-label`:`~

~https://codesandbox.io/s/react-daypicker-forked-2vfcoy?file=/src/App.tsx~

Edit: labelDay is not used anymore. See https://github.com/gpbl/react-day-picker/issues/1774#issuecomment-1596892784
